### DISCUSSION TITLE
Use a ballpark transform from ESRI to EPSG

### DISCUSSION THREAD

**Discussion by Ipickedausername** [opengisch/QField#4039] [Bugs]
Hi, 

I'm working from the Philippines. I decided to use a local CRS for my projects.

But on QField, any time I open a project, I have this same message appearing. I didn't understand what it really means and if it shows a real issue on the projects. 

Here is a screenshot of the message log:

![Screenshot_20230314-073317](https://user-images.githubusercontent.com/72577399/224972981-221a6a81-a558-45fa-876a-23b9d6d5fa8f.jpg)

Thanl you for your attention.

**Comment by lucienicolier** [comment]
Hi @Ipickedausername ,

Please have a look at this discussion [QField: "Preferred transform not available"](https://github.com/opengisch/QField/discussions/4115)
